include/cantera/zeroD/FlowReactor.h

```diff
@@ -48,6 +48,10 @@ class FlowReactor : public IdealGasReactor
     void evalDae(double t, double* y, double* ydot, double* residual) override;
 
     void getConstraints(double* constraints) override;
+    vector<size_t> steadyConstraints() const override {
+        throw CanteraError("FlowReactor::steadyConstraints",
+            "FlowReactor is not compatible with time-dependent steady-state solver.");
```

Comment:
I've not heard the phrase "time-dependent steady-state solver" before. Does that mean `advance_to_steady_state()` or the new solver here?